queries.sql

```diff
@@ -8,3 +8,11 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true; 
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg between 10.3 and 17.4;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts > 0;
```

Comment:
* Kindly write the correct query for this question(How many animals have never tried to escape?) currently, you are selecting animals with escape_attempts `greater than 0` please fix it 🙏